packages/core/webpack/prod.js

```diff
@@ -36,32 +37,55 @@ module.exports = [
   {
     name: "client",
     mode: "production",
-    devtool: "cheap-module-source-map",
     context: path.join(__dirname, "../src"),
     entry: {
-      app: [
+      client: [
+        "normalize.css/normalize.css",
+        "@blueprintjs/core/lib/css/blueprint.css",
+        "@blueprintjs/icons/lib/css/blueprint-icons.css",
+        "core-js/modules/es6.promise",
+        "core-js/modules/es6.array.iterator",
```

Comment:
I'd like to experiment with this in the future. Last week I had to investigate further on babel configs, and found there are ways to bundle these and other polyfills only if we're really using them. It's easier than we think.

Reply:
This got in here because Babel told me to (in order to get the new babel import plugin working): https://babeljs.io/docs/en/babel-plugin-syntax-dynamic-import#working-with-webpack-and-babelpreset-env

I still only barely understand this core-js stuff. I had tried to upgrade to using v3 at one point, but hit a bunch of problems and slowly backed away. Would love some help here in the future, because I think this is often tied to getting `regenerator-runtime` working, which would allow for async React class methods.